src/builtin.c

```diff
@@ -58,24 +58,23 @@ BINOPS
 
 static jv type_error(jv bad, const char* msg) {
   char errbuf[15];
-  jv err = jv_invalid_with_msg(jv_string_fmt("%s (%s) %s",
-                                             jv_kind_name(jv_get_kind(bad)),
-                                             jv_dump_string_trunc(jv_copy(bad), errbuf, sizeof(errbuf)),
+  const char *badkind = jv_kind_name(jv_get_kind(bad));
```

Comment:
So we're just trying to make `type_error()` fast?  :)

Reply:
I was reading the source code for `contains/1` and I found that unnecessary `jv_copy()`, then I looked for other unnecessary copies in `src/builtin.c` for fun, and I only found those that were easily avoidable. =)

Reply:
I rather liked having things like this as one C expression :) but you're right that unnecessary `jv_copy()`s are bad.